Pipfile

```diff
@@ -25,6 +25,7 @@ codacy-coverage = "*"
 pydocstyle = "*"
 radon = "*"
 xenon = "*"
+invoke = "*"
```

Comment:
If you install invoke in a non-pipenv environment as stated in the new documentation, do you still need it inside the virtualenv as well?

Reply:
Thanks for the suggestion, the invoke development dependency was removed from `Pipfile` and Pipfile.lock`.